anomalib/utils/callbacks/__init__.py

```diff
@@ -114,18 +114,36 @@ def get_callbacks(config: Union[ListConfig, DictConfig]) -> List[Callback]:
                     export_dir=os.path.join(config.project.path, "compressed"),
                 )
             )
-        if "openvino" in config.optimization and config.optimization.openvino.apply:
-            from .openvino import (  # pylint: disable=import-outside-toplevel
-                OpenVINOCallback,
+        if "openvino" in config.optimization.export_mode:
+            from .export import (  # pylint: disable=import-outside-toplevel
+                ExportCallback,
             )
 
+            logger.info("Setting model export to OpenVINO")
             callbacks.append(
-                OpenVINOCallback(
+                ExportCallback(
                     input_size=config.model.input_size,
-                    dirpath=os.path.join(config.project.path, "openvino"),
+                    dirpath=config.project.path,
                     filename="model",
+                    export_mode="openvino",
                 )
             )
+        elif "onnx" in config.optimization.export_mode:
+            from .export import (  # pylint: disable=import-outside-toplevel
+                ExportCallback,
+            )
+
+            logger.info("Setting model export to ONNX")
+            callbacks.append(
+                ExportCallback(
+                    input_size=config.model.input_size,
+                    dirpath=config.project.path,
+                    filename="model",
+                    export_mode="onnx",
+                )
+            )
+        else:
+            logger.info("Export option not found")
```

Comment:
Why is this info? Either we should throw an error or warn users that the option was incorrect and the training is proceeding without export callback.  Also, I prefer info to show what was passed `f"... option {config.optimization.export_mode} ..."` so that the users can see if they made a typo. Additionally, it would be nice to reiterate the options in the warning message.

Reply:
Addressed in: [27282f2](https://github.com/openvinotoolkit/anomalib/pull/509/commits/27282f21e1cb9454e2fbf30764b58bc459df84f1)